wrappers/Julia/CoolProp.jl

```diff
@@ -55,6 +55,7 @@ end
 
 # CoolProp::get_parameter_information_string
 function get_parameter_information_string(Key::AbstractString)
+message_buffer[1:4] = UInt8[108 111 110 103] # "long" written in UInt8
```

Comment:
Well, this is because get_parameter_information_string() wrapper in Julia was implemented to take only one argument. So, if it is to be kept (for backward compatibility?), it should return some meaningful information, which is hardcoded here.
It's another question if this wrapper should stay or if it should be replaced with another one with two arguments, following the underlying API.